.ops/jenkins/delete-nightly-snapshot.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+jfrog rt del --url "https://stardog.jfrog.io/stardog" \
+             --user ${artifactoryUsername} \
+             --password ${artifactoryPassword}  \
+             --quiet=true \
+             --spec=./ops/jenkins/snapshot/delete.spec || true
```

Comment:
If you haven't, you can try this locally and I strongly suggest you try it with the dry run option to see what it'll delete.

Reply:
Yes, I haven't done it yet. Will do it tomorrow. Will my own credentials work in this case (write permission needed)? I tried to publish it using scala build script it returned with authentication errors.

Reply:
I'm not sure but if you run into issues we can sort them out. Authentication errors to me sounds more like something wasn't setup correctly with the creds, not that they weren't authorized for certain permissions.

Reply:
I will look into it again, Paul.

Reply:
I had to make a little change in artifactory url in build.sbt. Both publishing to jfrog and deleting from it, are working fine.